stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -190,10 +190,45 @@ abstract type Algorithm end
 struct DivideAndConquer <: Algorithm end
 struct QRIteration <: Algorithm end
 
+# Pivoting strategies for matrix factorization algorithms.
 abstract type PivotingStrategy end
+
+"""
+    NoPivot
+
+Pivoting is not performed. Matrix factorizations such as the LU factorization
+may fail without pivoting.
+"""
 struct NoPivot <: PivotingStrategy end
+
+"""
+    RowNonZero
+
+First non-zero element in the remaining rows is chosen as the pivot element.
```

Comment:
```suggestion
First non-zero element in the remaining rows is chosen as the pivot element.

Beware that for floating-point matrices, the resulting algorithm is numerically unstable — this strategy
is mainly useful for comparison to hand calculations (which typically use this strategy) or for other
algebraic types (e.g. rational numbers) not susceptible to roundoff errors.   Otherwise, the default
`RowMaximum` pivoting strategy should be generally preferred.
```